entrypoint.sh

```diff
@@ -35,7 +35,7 @@ if [ "${UPGRADE_LINTERS_VERSION}" == "true" ]; then
   # Run only get_linter_help test methods
   pytest -v --durations=0 -k _get_linter_help megalinter/
   # Reinstall mkdocs-material because of broken dependency
-  pip3 install --upgrade "markdown==3.3.7" mike mkdocs-material "pymdown-extensions==9.11" "mkdocs-glightbox==0.3.2" mdx_truly_sane_lists jsonschema json-schema-for-humans giturlparse webpreview "github-dependents-info==0.10.0"
+  pip3 install --upgrade -r .config/python/dev/requirements.txt
```

Comment:
This however might not work, since I don't think that that file is copied inside the released docker images. The entrypoint.sh script is what is ran when running the docker images. Should we either include the dependencies file, or find a way to keep the deps needed (and not installed by default, like node's dev dependencies), or keep it manually in sync?

Reply:
Reverted (with changed versions) and added note to keep in sync.